internal/util/crypto.go

```diff
@@ -255,6 +255,16 @@ func OpenEncryptedVolume(ctx context.Context, devicePath, mapperFile, passphrase
 	return err
 }
 
+// ResizeEncryptedVolume resizes encrypted volume so that it can be used by the client.
+func ResizeEncryptedVolume(ctx context.Context, mapperFile string) error {
+	DebugLog(ctx, "Resizing LUKS device %s", mapperFile)
+	_, stderr, err := LuksResize(mapperFile)
+	if err != nil {
+		WarningLog(ctx, "failed to resize LUKS device %s: %s", mapperFile, stderr)
```

Comment:
not sure about places this should be an error log, not the warning log.

Reply:
This was inspired from OpenEncryptedVolume() function above.

Reply:
Can you please change this to error log. OpenEncryptedVolume() can be corrected later

Reply:
On it now.

Reply:
Done with both.

Reply:
`OpenEncryptedVolume()` was moved to ErrorLog, this is still WarningLog?

Reply:
correct